.github/workflows/docs.yaml

```diff
@@ -0,0 +1,78 @@
+name: Build Docs for the latest
+
+on:
+  workflow_dispatch: # run on request (no need for PR)
+  push:
+    branches:
+      - develop
```

Comment:
Is there any way to test docs before this v2 is replaced by develop?

Reply:
@harimkang I've opened a PR for that. #3091

Reply:
Thanks!